contracts/ProtocolReserve/RiskFundV2.sol

```diff
@@ -75,25 +80,27 @@ contract RiskFundV2 is AccessControlledV8, RiskFundV2Storage, IRiskFund {
     /// @param shortfallContractAddress_ Address of the auction contract
     /// @custom:event ShortfallContractUpdated emit on success
     /// @custom:error ZeroAddressNotAllowed is thrown when shortfall contract address is zero
+    /// @custom:access Only Governance
     function setShortfallContractAddress(address shortfallContractAddress_) external onlyOwner {
         ensureNonzeroAddress(shortfallContractAddress_);
         emit ShortfallContractUpdated(shortfall, shortfallContractAddress_);
         shortfall = shortfallContractAddress_;
     }
 
-    /// @dev Transfer tokens for auction
+    /// @dev Transfer tokens for auction to bidder address
     /// @param comptroller Comptroller of the pool
-    /// @param bidder Amount transferred to bidder address
-    /// @param amount Amount to be transferred to auction contract
-    /// @return Number reserved tokens.
+    /// @param amount Amount to be transferred to the bidder
+    /// @return Amount of tokens transferred to the bidder
```

Comment:
After the changes made in https://github.com/VenusProtocol/protocol-reserve/pull/29/commits/cf5e3713edb5c4a4f574881cb9f82d375625237e, the target of the transfer is the shortfall contract, not the bidder. We should update this comment in the context of VPB-12